Source/monster.cpp

```diff
@@ -11,6 +11,12 @@
 
 namespace devilution {
 
+#define NIGHTMARE_TO_HIT_BONUS 85
+#define HELL_TO_HIT_BONUS 120
+
+#define NIGHTMARE_AC_BONUS 50
+#define HELL_AC_BONUS 80

```

Comment:
Same as the other one, I always thought constants and defines were to be placed in the header, not the cpp. Was I incorrect?

Reply:
same